scripts/apu_fw_updater_opnsense.sh

```diff
@@ -17,7 +17,7 @@
 TYPE="apu2"
 
 # Version of firmware
-VERSION="4.15.0.3"
+VERSION="4.17.0.3"
 
 # Do not edit after this line
 SRC="https://3mdeb.com/open-source-firmware/pcengines/${TYPE}/"
```

Comment:
While at it, maybe change the URL to dl.3mdeb.com?

Reply:
Done